internal/metrics/metrics.go

```diff
@@ -30,6 +30,14 @@ type AppProjectMetrics struct {
 	AppProjectWatcherMetrics
 }
 
+type InformerMetrics struct {
+	ResourcesListed *prometheus.GaugeVec
+	ListDuration    *prometheus.GaugeVec
+	AddDuration     *prometheus.GaugeVec
+	UpdateDuration  *prometheus.GaugeVec
+	DeleteDuration  *prometheus.GaugeVec
+}
```

Comment:
This slipped in, but is not used right now (except in some tests I think). It's not yet finished, either.